src/main/java/com/squareup/javawriter/builders/CodeWriter.java

```diff
@@ -68,6 +77,24 @@ public CodeWriter popVisibleType(TypeName typeName) {
     return this;
   }
 
+  public void emitAnnotations(ImmutableList<AnnotationSpec> annotations) {
+    if (annotations.equals(OVERRIDE_ONLY)) {
```

Comment:
Despite my love for compactness, this seems like a needless special case. Especially since we often see it push lines past 100 chars.